fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java

```diff
@@ -189,7 +189,8 @@ public Status createTabletsForRestore(int tabletNum, MaterializedIndex index, Gl
         List<Long> shardIds = null;
         try {
             // Ignore the parameter replicationNum
-            shardIds = globalStateMgr.getStarOSAgent().createShards(tabletNum, fsInfo, cacheInfo, shardGroupId, properties);
+            shardIds = globalStateMgr.getStarOSAgent().createShards(tabletNum, fsInfo, cacheInfo, shardGroupId, null, properties,
```

Comment:
it looks a little weird that the interface without workerGroupId is removed from StarOSAgent, but then in the caller, the workerGroupId is specified with a hard coded value. Then why remove the interface?

Reply:
I think we should not let StarOSAgent to decide which workerGroupId is used. An interface without workerGroupId means StarOSAgent will hard coded this value.

Reply:
then the StarOSAgent.DEFAULT_WORKER_GROUP_ID should not be used here, should be replaced by the `defaultWarehouse.getWorkerGroupId()` or something similar.

It is intended to let StarOSAgent interface not determine the workerGroupId and then use a constant defined in StarOSAgent, sounds like StarOSAgent does know everything, but pretend to know nothing.